src/Data/Entity/Login.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace App\Data\Entity;
+
+use App\Data\Authenticator;
+use App\Data\Entity;
+
+class Login extends Entity
+{
+
+    protected $transformer = 'App\Data\Transformer\LoginTransformer';
```

Comment:
That will work, I was just thinking about other formats. Thought too hard about it.